src/renderer/api/endpoints/helm-charts.api.ts

```diff
@@ -5,7 +5,7 @@ import { autobind } from "../../utils";
 
 interface IHelmChartList {
   [repo: string]: {
-    [name: string]: HelmChart;
+    [name: string]: HelmChart[];
   };
 }
```

Comment:
This sub type seems to be used in multiple places. Should it just be declared here and imported above?

```suggestion
export type RepoHelmChartList = Record<string, HelmChart[]>;
export type HelmChartList = Record<string, RepoHelmChartList>;
```

Reply:
Good change, keeping your types. Thanks!

Reply:
Maybe this `type HelmChartList = Record<string, HelmChart | HelmChart[]>`?